### DISCUSSION TITLE
Server staff bio page

### DISCUSSION THREAD

**Discussion by aedwardg** [CodecademyCommunity/CodecademyCommunity.github.io#11] [Ideas]
What does everyone think about having a server staff bio page?

I think some community members might find it cool to know a little bit more about our staff members.  This could also serve as a place to include descriptions of the various staff roles.

Of course, we don't need to add personal info like real names, pics, GH handles, etc. if we don't want to, but at least a Discord handle and short blurb about each of us might be nice.

**Comment by Vic-ST** [comment]
+1, it'd be a great way to keep track of everyone on the team as well

**Comment by Victoria-DR** [comment]
Server staff page was created for the community website. Issue: https://github.com/CodecademyCommunity/CodecademyCommunity.github.io/issues/38.
